### ISSUE TITLE
Prevent deleting posts with coauthors

### ISSUE THREAD

**Issue by teceler** [Marri/glowfic#800]
Deletion request was suggested and seems reasonable, but maybe just block it until that's implemented?

**Comment by Throne3d** [comment]
I don't think we should block it – there are legitimate use cases, whereas there have only been limited cases of abuse – but we should check in with coauthors before acting on a request to delete a post.

**Comment by Throne3d** [comment]
Looks like currently you can't even delete a post with replies, since it destroys the post authors before the replies, and then reply callbacks fail.

**Comment by Throne3d** [comment]
Reviewing this, blocking it initially and requiring mod intervention seems legit for now. I'll see if I can add a test that it doesn't go through and suggests for you to call a mod.
